sources/engine/Stride.Rendering/Rendering/Images/ColorTransforms/ToneMap/ToneMapACESOperatorShader.sdsl

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Stride contributors (https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
Should remove Silicon Studio Corp. for new files.
It applies to many other locations in this commit.

Reply:
Never mind, let's fix that in `master` since it's just a backport.